doc/api/errors.md

```diff
@@ -2115,6 +2179,16 @@ removed: v10.0.0
 Used when a failure occurs sending an individual frame on the HTTP/2
 session.
 
+<a id="ERR_QUIC_TLS13_REQUIRED"></a>
+### ERR_QUIC_TLS13_REQUIRED
+<!-- YAML
```

Comment:
Should this be a TBD

Reply:
PR: https://github.com/nodejs/quic/pull/228